.editorconfig

```diff
@@ -0,0 +1,14 @@
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.java]
+indent_size = 4
```

Comment:
The guideline also use 2 spaces only (https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation) maybe that's better too ?